README.md

```diff
@@ -16,864 +16,26 @@
-
-**models/Model.js**
-
-```js
-import { Model as BaseModel } from 'vue-api-query'
-
-export default class Model extends BaseModel {
-
-  parameterNames () {
-    return {
-      include: 'include_custom',
-      filter: 'filter_custom',
-      sort: 'sort_custom',
-      fields: 'fields_custom',
-      append: 'append_custom',
-      page: 'page_custom',
-      limit: 'limit_custom'
-    }
-  }
-}
-```
-
-# Response from backend
-
-This package automatically handles the response from backend and convert it into an instance of a such Model.
-
-## Single object
-
-If your backend responds with the following response...
-
-```js
-// Note: the response data is the root element with no 'data' wrapper
-{
-  id: 1,
-  firstname: 'John',
-  lastname: 'Doe',
-  age: 25
-}
-```
-
-Then using `find()` and `first()` will work as expected and will hydrate the response into the `User` Model.
-
-```js
-let user = await User.find(1)
-
-// or
-
-let user = await User.first()
-
-// will work - an instance of User was created from response
-
-user.makeBirthday()
-```
-
-However, if the backend sends a response like this...
-
-```js
-// Note: the response is wrapped with 'data' attribute...
-data: {
-  {
-    id: 1,
-    firstname: 'John',
-    lastname: 'Doe',
-    age: 25
-  }
-}
-```
-
-...then the above would fail. If your backend wraps single objects with a data attribute too then you should use the fetch method of find (which is `$find()`) instead to automatically hydrate the Model with the response data:
-
-```js
-let user = await User.$find(1)
-
-// or
-
-let user = await User.$first()
-
-// will work, because an instance of User was created from response
-
-user.makeBirthday()
-```
-
-This **WILL NOT** be converted into `User` model, because the main data is not the root element or it is not wrapped by `data` attribute.
-
-```js
-user: {  
-    id: 1,
-    firstname: 'John',
-    lastname: 'Doe',
-    age: 25
-}
-```
-
-## Array of objects
-
-An array of items from backend would be converted in the same way, **ONLY** if it responds in these formats:
-
-```js
-let users = await User.get()
-```
-
-```js
-// works - array of object is the root element
-[
-  {
-    id: 1,
-    firstname: 'John',
-    lastname: 'Doe',
-    age: 25
-  },
-  {
-    id: 2,
-    firstname: 'Mary',
-    lastname: 'Doe',
-    age: 22
-  }
-]
-```
-
-```js
-// works - `data` exists in the root and contains the array of objects 
-{
-  data: [
-    {
-      id: 1,
-      firstname: 'John',
-      lastname: 'Doe',
-      age: 25
-      },
-    {
-      id: 2,
-      firstname: 'Mary',
-      lastname: 'Doe',
-      age: 22
-    }
-  ],
-  someField: '',
-  anotherOne: '',  
-}
-
-// Normally you would handle the response like this
-
-let response = User.get()
-let users = response.data
-
-
-// or like this
-
-const { data } = User.get()
-let users  = data
+<p align="center">
```

Comment:
This README file already have HTML tags at the top because of labels. I just added "read documentation" based on [nuxt auth readme](https://github.com/nuxt-community/auth-module/blob/master/README.md). But I agree that it's kind of invisible here.

Language change will be made at the website, there is no need to show it here.

> Please always use markdown syntax for links: [description] (url)

Yes, I know.

Reply:
Fixed in c4c87c0b0de8880c32d859d14e80607fdef6e803